x-pack/plugins/observability/public/utils/observability_index_patterns.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataPublicPluginStart, IndexPattern } from '../../../../../src/plugins/data/public';
+
+export type DataType = 'synthetics' | 'apm' | 'logs' | 'metrics' | 'rum';
+
+const indexPatternList: Record<DataType, string> = {
+  synthetics: 'synthetics_static_index_pattern_id',
+  apm: 'apm_static_index_pattern_id',
+  rum: 'apm_static_index_pattern_id',
+  logs: 'logs_static_index_pattern_id',
+  metrics: 'metrics_static_index_pattern_id',
+};
+
+const appToPatternMap: Record<DataType, string> = {
+  synthetics: 'heartbeat-*',
```

Comment:
Will this need to include `synthetics-*` down the line?

Reply:
you are right, actually instead of hardcoding, we will need to expose a handler from each app, which will return index settings from saved objects. for uptime case, the index setting we define on settings page.